akd/src/append_only_zks.rs

```diff
@@ -44,6 +44,12 @@ pub struct Azks {
     pub num_nodes: u64, // The size of the tree
 }
 
+impl SizeOf for Azks {
+    fn size_of(&self) -> usize {
+        std::mem::size_of::<u64>() * 2
```

Comment:
Why not `mem::size_of::<Azks>()` and similarly for other structs?

Reply:
It can't measure dynamic structs which are on the heap, like `Vec<_>`'s. It'll give a constant size for all vectors since the `Vec` struct itself is backed by a `RawVec` which is just a pointer into memory. We need our own size measurement tool to properly multiply by the size of the vector and measure the size of the contained elements. I use the standard one for primitive types, but for safety stay away from it for our structs.